client/me/purchases/pre-cancellation-dialog/index.tsx

```diff
@@ -238,16 +247,25 @@ export const PreCancellationDialog = ( {
 							} ) }
 							align="left"
 						/>
+						<h2>{ subTitle }</h2>
 						<FeaturesList
 							productSlug={ productSlug }
-							subTitle={ subTitle }
 							domainFeature={ domainFeature }
 							isPurchaseRefundable={ isPurchaseRefundable }
 							isPurchaseAutoRenewing={ isPurchaseAutoRenewing }
 						/>
-						<RenderFooterText purchase={ purchase } />
 					</div>
+					{ shouldUseSiteThumbnail && (
+						<div className="pre-cancellation-dialog__grid-colmn">
+							<SiteScreenshot
+								className="pre-cancellation-dialog__site-screenshot"
+								site={ site }
+								alt={ siteName }
```

Comment:
Using `siteName` as the alt text here is an accessibility issue since it doesn't convey that it's a screenshot of the site. We can find plenty of lacking meaningful alt text issues across calypso, but here is a good example: https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/checkout/checkout-thank-you/premium-plan-details.jsx#L39

What do you think if we put something like "The screenshot of the site: ${siteName}"

Reply:
this is fixed with `alt={ 'The screenshot of the site: ' + siteName }`

Reply:
Thanks for the fix. However, one thing to add is that it should also be translated like [this example](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/checkout/checkout-thank-you/premium-plan-details.jsx#L39). I don't expect this to have any formal copy review, so let's just go for it :)

Reply:
@southp  I added the `translate()` function